packages/sdk/tests/index.test.ts

```diff
@@ -64,14 +63,6 @@ describe('sdk', async () => {
       }
     });
 
-    it('errors if signer is registered without a provider', () => {
```

Comment:
where did these go?

Reply:
it no longer errors because it has a default provider instantiated